### PR TITLE
Don't shorten reported experiment time when data is incomplete.

### PR DESCRIPTION
**Pull request by lszekeres**
Fixes https://github.com/google/fuzzbench/issues/482

### CODE REVIEW

**Comment by lszekeres** [comment]
(Will fix tests if we want to go forward with this)

File: analysis/data_utils.py [hunk 1]
```diff
@@ -100,17 +100,18 @@ def filter_max_time(experiment_df, max_time):
...
 
 def get_benchmark_snapshot(benchmark_df,
                            threshold=_DEFAULT_BENCHMARK_SAMPLE_NUM_THRESHOLD):
-    """Finds the latest time where 80% of the trials were still running. In most
-    cases, this is the end of the experiment. In this case, we won't consider
-    the <20% of the trials that ended early for our analysis. If more than 20%
-    of the trials ended early, it's better to pick an earlier snapshot time.
-    The 80% can be overridden using the |threshold| argument. E.g., to find the
-    latest time where each trials were running, set |threshold| to 1.0.
+    """Finds the latest time where say 80% of the trials were still running (when
```
Comment (jonathanmetzman) [inline analysis/data_utils.py:108]: I'm not sure the 80% part of the comment makes sense anymore.

**Comment by jonathanmetzman** [comment]
I think we should go through with this.

**Comment by jonathanmetzman** [comment]
Ping, I think this got forgotten about between my delay in reviewing it and the holiday.

File: analysis/data_utils.py (hunk 1 above)
Comment (lszekeres) [reply analysis/data_utils.py]: Done

**Pull request merged**